libs/server-sdk/src/data_systems/lazy_load/lazy_load_system.cpp

```diff
@@ -1,22 +1,16 @@
 #include "lazy_load_system.hpp"
```

Comment:
Linter actually got around to this file since the PR is small enough, so fixed the lints here.